README.md

```diff
@@ -19,6 +20,12 @@ Install CyFi with pip:
 pip install cyfi
 ```
 
+If you're on a Mac, you should install from conda-forge instead:
```

Comment:
I think this is an oversimplification and not exactly right.

- People not on Macs might want to install from conda-forge if they're just generally using conda. The thing we want to say is "CyFi is also available with conda."
- Not everyone on a Mac will have the error from #134. We know that I don't have this problem on an Intel Mac. We also haven't heard from anyone who is using an M2 or M3 Mac. 

I think it may make more sense to just put a notice that specifically describes there being a known issue on M1 Macs that with LightGBM and libomp, and that this can be worked around by installing with conda. 

If you don't explain why you're telling certain people to install from conda, then this ends up being a magic incantation that people don't really understand why/when they need it. LightGBM could even fix the underlying problem (by adding the additional locations libomp could be installed to on M1 Macs).

Reply:
I had a feeling you'd have this take which is in part why I requested the review from you. I agree with giving people more information but at the same time, the docs are intentionally as non-technical as possible given the expected user base. I'll take another stab at it. Also: thanks for the info about intel macs, I thought there was an issue there as well but I guess that was a separate lightGBM issue that katie had run into on another project.

Reply:
Maybe stick something at the beginning of the "Install" section, like:

> If you are on a Mac, we recommend installing with conda (see second option) to avoid a potential error. More details can be found [here](https://github.com/drivendataorg/cyfi/issues/114).

You could use one of the admonition boxes to make it more prominent. https://gist.github.com/jayqi/de4fbef3082b7cebead384be365485fb

Reply:
I took another pass at this, let me know if you think this has sufficient info now or if more is needed to let people make informed choices